app/models/user.rb

```diff
@@ -7,5 +7,5 @@ class User < ApplicationRecord
          :recoverable, :rememberable, :validatable
 
   # Active Record Associations
-  has_many :appointments
+  has_many :appointments, dependent: :destroy
```

Comment:
Faz com que os objetos relacionados sejam deletados juntamente quando o objeto for removido.

Reply:
Está bacana 👍🏽